### ISSUE TITLE
Add support for extra information in Chercker

### ISSUE THREAD

**Issue by prudhomm** [feelpp/feelpp#993]
Checker allows to verify that simulation provides expected results. 
These results are stored in a json file that 
 * could be embedded into a bigger json file, eg a simulation report. 
 * could be used for articles / publications
 
It would be nice to be able to provide extra information such as
 * number of dof if it make sense
 * number of non-zero entries if it makes sense

NOTE that like `h` these entries are tricky because they depend on the mesh

if we mix this with #943 eg timing to obtain the results (time to {init,assemble,solve}) + machine info (#942)+ feelpp config + number of cores and we have a nice benchmarking system #939

**Comment by prudhomm** [comment]
we could add some extra info, eg 
```
{ { "name_displayed", variable, "short_description" } }
```

**Comment by lsala** [comment]
it could be useful in same cases, but for others maybe just the name is self-explaining

**Comment by prudhomm** [comment]
@lsala indeed, in the case of @romainhild `u` is not very explicit we could add a description in taht case eg `L2 norm u` or `L2 norm flux `

**Comment by romainhild** [comment]
Maybe the easiest solution is to change the name to `L2 flux`, it would allow to keep a `std::map<std::string,double>`.
But it could be useful if we need more info, for example the range on which we compute the error, or maybe for the validation.

**Comment by prudhomm** [comment]
> But it could be useful if we need more info, for example the range on which we compute the error, or maybe for the validation.

What do you mean by : the range ...

**Comment by romainhild** [comment]
well for example if we compute the error only on a part of the domain. For example in the case of the geometrical order.
